api/envoy/config/filter/network/client_ssl_auth/v2/client_ssl_auth.proto

```diff
@@ -30,6 +30,9 @@ message ClientSSLAuth {
   // 0-refresh_delay_ms milliseconds.
   google.protobuf.Duration refresh_delay = 3 [(gogoproto.stdduration) = true];
 
+  // Time in milliseconds for an authentication service request to timeout. Default is 1000 (1s).
+  google.protobuf.Duration request_timeout = 5 [(gogoproto.stdduration) = true];
```

Comment:
TBH I would revert all the changes for this filter until someone asks for this.

Reply:
Addressed in 0ecfc41.